pkg/apis/storagepool/config/cns.vmware.com_storagepools.yaml

```diff
@@ -1,10 +1,10 @@
 
 ---
-apiVersion: apiextensions.k8s.io/v1
+apiVersion: apiextensions.k8s.io/v1beta1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.6.2
```

Comment:
why do we need to change version from `v0.6.2` to `v0.2.5`?
Also we are moving back to `apiextensions.k8s.io/v1beta1`from `apiextensions.k8s.io/v1`?

Reply:
Good catch, I had an old `controller-gen`. Fixed now.